### PR TITLE
Add Heroku and Fastly logos to footer

### LINKED ISSUE #2910: Add the Heroku logo to our footer
As part of switching our website to Heroku hosting (with Fastly CDN), we want to add sponsor logos to our footer thanking Heroku (and Fastly in the near future).

@wifelette and I have discussed adding those two logos to the right side of the footer (I've put a rough draft below, but it is definitely rough).  We need someone to champion this project and help it get finished (I don't think I'll have time). Not all the items below have to be done by the same person, but getting the ball rolling on the design is the where this issue has gotten stuck.

This project will be done when:
- [ ] the design has been iterated on (screenshots posted here are probably enough to help the conversation along)
- [ ] @wifelette has signed off on the design
- [ ] the design has been implemented for
  - [ ] website repo (this repo)
  - [ ] guides repo (https://github.com/emberjs/guides)
  - [ ] the api repo (https://github.com/ember-learn/ember-api-docs)

Heroku assets/info:
- [Brand Guidelines](https://www.dropbox.com/s/m6hb2x4s0uog1gg/guidelines.pdf?dl=0)
- [All logo options](https://www.dropbox.com/sh/cd0h7jzalg1m7id/AACI8rneaxZT0VnXhRotMW7ra?dl=0)
- [Logo we likely want to use, see in below screenshot](https://www.dropbox.com/sh/cd0h7jzalg1m7id/AAAmEicKss8ojSd_e3yDofhAa/SVG%20(Vector)?dl=0&preview=heroku-logo-stroke-purple.svg)

Fastly assets/info:
- I don't see any when browsing their site, please comment if you see some and we can update this description

Rough draft:
screen shot 2017-06-01 at 12 32 24 pm

### PR DESCRIPTION
**Pull request by gpxl**
screen shot 2017-06-02 at 12 45 08 pm

resolves #2910

### CODE REVIEW

**Comment by hone** [comment]
@acorncom looks good to me, let me pass it along to my marketing team since I'm not the brand person at heroku.

**Comment by wifelette** [comment]
Can we adjust so the full height of the two logos are the same? In this case likely means scaling down the Heroku logo a bit. If that makes it look too small they can likely send us their horizontal version so the two logos look more equitable.

Also, @gpxl, did you work with @amyrlam on this? I know you're both at Fastly so assuming so, but just curious :p

**Comment by gpxl** [comment]
@wifelette The images are (dimensionally) the same height although visually they don't appear to be. Here's a comp with the Heroku logo adjusted down a bit and a version with the horizontal logo. The Horizontal logos look better IMO but I'll update with whichever folks prefer.

ember-logos

@amyrlam and I chatted about this & I ran with it, with her blessing :)

**Comment by itchymutt** [comment]
Heroku design's 2cents: While dimensionally the same, it would look better if you align the fastly logo's baseline. See attached:

screenshot 2017-06-02 15 35 26

If you went with the horizontal logo, then I'd prefer that the logo increase in size so that HEROKU falls neatly within the top & bottom of the 's' in the fastly logo.

cc @hone

**Comment by gpxl** [comment]
Thanks for the feedback @itchymutt! PR has been updated per recommendation.

ember-logos-revised

**Comment by crcastle** [comment]
Hi folks.  I just chatted with @itchymutt (leads Heroku design) and we ended up at recommending logo uses for this purpose go with what's specified on page 16 of [this PDF](https://heroku.com/brand) (or page 12 per table of contents).  So two options:

1. Simply change the "Hosted by" text to "Built on".
2. Replace the image and "Hosted by:" text with [this image](http://hrku.co/built-on-heroku).

Super sorry for the run around on this.  Ember is one of the first (of hopefully many) projects that we're sponsoring and logo-ing up like this!

**Comment by gpxl** [comment]
@crcastle No problem.

Although, it looks like I messed up the git history on this PR. I'm not sure how to rectify without digging a deeper hole. Any git expert want to help?

**Comment by wifelette** [comment]
Hi @crcastle. Can you explain the request a bit, to see if there's something better we could come up with? This is butting up against two constraints:

1. We need it to be a consistent style that will apply to all kinds of potentially donated services (for example Fastly now too of course), so the "X by" needs to be copy from our end versus in the image
2. "Built on" is honestly overly broad, and I don't think is a phrasing we could get comfortable with. So _simply_ isn't quite right for that request ;) Hosted by is indeed _accurate_, so we like that, but if there's some suggestion similar to _that_ we'd be happy to think through it!

Also, this doesn't have to block merging, but we should get this moved to our usual format, where the inline code just uses Ruby to pull from a Yaml file with the data. Fields in this case would be something like `service text` (a la _hosting_, _CDN_, etc), `image` and `url`. That way, like all other things, it's smooth sailing later when we want to add other things down there.

**Comment by crcastle** [comment]
Hi @wifelette. Thanks for the feedback.  I admit that I'm not psyched about either "built on" or "hosted by".  Built on is vague and broad.  But hosted by is too narrow -- but I could be wrong here because I don't know exactly what Heroku functionality the emberjs team is using.  For example, while Heroku does provide hosting in the sense of "here's a container to put your app in", we also build the environment the app runs in, define and manage the build process, monitor the server process, route traffic when the app scales, provision and actively manage postgres, provide [CI](https://www.heroku.com/continuous-integration) and [CD](https://www.heroku.com/continuous-delivery) for the app, and many other things.

I do not know the right word that captures all this, but I realize that's a marketing problem for us to figure out, not you.  So I'm fine going with "hosted by" if you're not comfortable with "built on".

Mostly we just want people to know that Heroku helps to make it easier to build Ember sites, and to demonstrate that, the Ember team uses Heroku for this site and the docs site.

File: source/_footer.html.erb [hunk 1]
```diff
@@ -16,18 +16,29 @@
...
-      <a href="https://github.com/emberjs/ember.js" title="GitHub">
-        <i class="icon-github"></i>
-      </a>
-      <a href="https://plus.google.com/communities/106387049790387471205" title="Google+">
-        <i class="icon-gplus"></i>
-      </a>
+    <div class="footer-contributions">
+      <div class="contributor">
+        <p>Built on:</p>
+        <a href="https://www.heroku.com"><img src="/images/logos/vendors/heroku-logo.svg" class="logo-heroku"></a>
```
Comment (Dhaulagiri) [inline source/_footer.html.erb]: I would suggest this link to either http://heroku.com/emberjs or http://heroku.com/home.  @crcastle do you have a preference?

**Comment by wifelette** [comment]
@crcastle Let's ship with Hosted by, but we're here and totally open to revisit when better options come up on your end :) 

> Mostly we just want people to know that Heroku helps to make it easier to build Ember sites, and to demonstrate that, the Ember team uses Heroku for this site and the docs site.

+1, we're proud (and grateful) Heroku users!

Earlier in the thread someone mentioned the logos maybe being too large, so I just want to address that here for posterity: they're probably on the generous side for this sort of thing, but I'm comfortable with it for now :) Just note that should there be others to add to the list later, the sizing will no doubt be shrunk. May as well be generous while we can though!

**Comment by wifelette** [comment]
Ping @crcastle RE: query on where the logo should link to! Also needs to be swapped back to Hosted by but then we should be good to merge :)

**Comment by crcastle** [comment]
Thanks for the ping @wifelette.  I'll update PR shortly with the link and "Hosted by" text.  I'd like to link to heroku.com/emberjs.  I met @locks at JSConf EU this year and he had some ideas on improving that page.

@locks I think you suggested changing heroku.com/emberjs to use super-rentals as the demo app to deploy on Heroku.  Anything else you'd like to see there?  Links to specific ember docs or tutorials?  Thanks!

**Comment by locks** [comment]
Thank you everyone for the good work! 😀 Let's get this to CI and deployed!
Heya @crcastle! I've been busy with a new job, but I'm definitely still up for chatting about it. We could pull @hone into the mix and get started on documenting the buildpack in the page :)

**Pull request merged**